.circleci/config.yml

```diff
@@ -400,6 +406,27 @@ commands:
       - store_artifacts:
           path: /tmp/junit
 
+  halt-playwright-check:
```

Comment:
Using this check in Playwright jobs to ensure Playwright jobs do not run for Staging, Prod and UNKNOWN env.